assignment10/controllers/projectController.js

```diff
@@ -0,0 +1,59 @@
+var mongoose = require('mongoose'), 
+  project = mongoose.model('projects');
+
+exports.allProjects = function(req, res){
+  	project.find({}, {'_id' : 0, 'manager' : 0, 'developers' : 0},(err, result) => {
+	    if (err != undefined) {
+    		res.send(err);
+    	}
+    	else {
+    		res.send(result);
+    	}
+    });
+};
+
+exports.addProject = function(req, res){
+    console.log(req.body);
+    var newProject = new project(req.body);
+    newProject.save(function(err) {
+        if (err) res.send(err);
+        else {
+          res.send('Succesfully added project');
+        }
+    });
+};
+
+exports.getDetails =function(req, res){
+	var title = req.params.title;
+  	console.log(title);
+  	project.findOne({'title' : title}, {'_id': 0}).populate('manager').populate('developers').exec(function(err,result){
+    	if (err) res.send(err);
+    	else if (result == undefined)
+    	{
+    		res.send('Invalid title');
+    	}  
+      	else {
+        	res.send(result);
+        }
+  	});
+};
+
+exports.removeProject =function(req, res){
```

Comment:
@abansal26 Same mistakes - 
- Indentation
- Space after function() and equal = sign.

Reply:
added space. Commit ID - 0836b72b25535718a21f16fe6296d2aea4d7e10d